packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,202 +24,274 @@ _ref:
-            - `types: object`: An object to customize and add block types.
+            - `plugins: object[]`: An array of `plugin` objects to customize and add block types.
             - `menus: object[]`: An array of menu objects.
             - `pages: object[]`: An array of page objects.
 
-            # Config
-
-            The config object has the following properties:
+            Pages are made up of blocks. Blocks have the following basic schema:
+            - `id: string`: __Required__ - A unique identifier for a block.
+            - `type: string`: __Required__ - This is the block type identifier and defines what block to used.
+            - `properties: object`: All the settings passed to a block component.
+            - `blocks: array`: An array of blocks to render within this block.
 
-            - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
-            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
+            Find the more detailed block schema [here](/blocks).
 
-      - id: alert1
+      - id: alert2
         type: Alert
         properties:
-          type: warning
+          type: info
           showIcon: false
-          message: Init page is an experimental feature, that may disappear in future releases as well as the flag itself can be changed. Use at your own risk.
+          message: |
+            <h3> YAML file extensions </h3>
+            <p> Both files with the <code>.yaml</code> and <code>.yml</code> file extensions are supported as YAML files. </p>
+            <h3> JSON instead of YAML </h3>
+            <p> Since you can reference JSON files, you can build your app using JSON instead of YAML files. The <code>lowdefy.yaml</code> file needs to be a YAML file, but all other configuration can be in referenced JSON files. It also makes sense to use JSON instead of YAML if you are generating configuration using code. </p>
 
-      - id: md2
+      - id: md3
         type: MarkdownWithCode
         properties:
           content: |
+            # Config
+
+            The config object has the following properties:
+
+            - `basePath: string`: Set the base path to serve the Lowdefy application from. This will route all pages under `https://example.com/<base-path>/<page-id>` instead of the default `https://example.com/<page-id>`. The basePath value must start with "/".
+            - `homePageId: string`: The pageId of the page that should be loaded when a user loads the app without a pageId in the url route. This is the page that is loaded when you navigate to `yourdomain.com`.
```

Comment:
We need to add theme less variables here.

@Gervwyk we need to decide on a schema for user defined style files, theme variables, appendHead etc. Not sure about the `app` and `config` sections.

Reply:
Yes. We need to have a session to finalise the schema on these.